Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/blueprint-manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "display-name": "Annotations Framework",
   "system-name": "Annotations",
-  "description": "(Preview) Use the .NET Lambda Annotations framework to write Lambda Functions.",
+  "description": "Use the .NET Lambda Annotations framework to write Lambda Functions.",
```

Comment:
* Rename this to `Annotations Framework Sample`
* Edit the description to here to show how this is a sample to demonstrate the features of the framework.
* Show off the DI by adding an `ICalculatorService` and registering it in the DI. Then updated `Startup` class to register it.
* In the test project you could create a Mock `ICalculatorService` to show how with DI you can mocking the services.

Reply:
Done. 

I only injected the `ICalculatorService` into the constructor since that felt more realistic, but we could also do a `[FromServices]` for the sake of it.